BruteShark/BruteSharkCli/BruteSharkCli.cs

```diff
@@ -122,6 +131,7 @@ private void UpdateAnalyzingStatus()
                 Console.WriteLine($"\r[+] TCP Sessions Analyzed: {_tcpSessionsCount}" + $" UDP Streams Analyzed: {_udpStreamsCount}");
                 Console.WriteLine($"\r[+] Passwords Found: {_passwords.Count}");
                 Console.WriteLine($"\r[+] Hashes Found: {_hashes.Count}");
+                Console.WriteLine($"\r[+] Network Connections Found: {_connections.Count}");
```

Comment:
You have added a new line, therefore should be Console.CursorTop - 5
also change to Console.CursorTop + 5 on line 236.
After fixing this the printing will look better:
![image](https://user-images.githubusercontent.com/18364847/100027954-7e5a8b80-2df6-11eb-94c2-9af0351c1f47.png)

Reply:
fix